config/catalyst-dev.json

```diff
@@ -99,7 +99,9 @@
         "connector": "livepeer-catalyst-api",
         "port": "7979",
         "source-output":"/tmp/livepeer/source",
-        "vod-pipeline-strategy":"catalyst"
+        "vod-pipeline-strategy":"catalyst",
+        "api-server":"http://127.0.0.1:3004",
```

Comment:
Are you sure this doesn't break the non-Studio setup? We check this variable for whether we should enable mapic, setting up the PUSH_REWRITE trigger and whatnot https://github.com/livepeer/catalyst-api/blob/9b77aefdc7ec2c00a38f812c6024d2893400c773/config/cli.go#L76

Reply:
Good point. I actually checked it and `make dev` worked fine :)

Reply:
Should I check anything else than vod upload endpoint?

Reply:
Try sending in a livestream, I assume it will break for lack of a Studio endpoint

Reply:
Like push `rtmp://127.0.0.1/live/stream+foo` and playback `http://localhost:8080/hls/stream+foo/index.m3u8`

Reply:
Push to `rtmp://127.0.0.1/live/stream+foo` doesn't work even before this PR. Does it work for you?

Reply:
Ah my bad, didn't realize this config has "video" and not "stream" enabled. Works for me though with
```
docker run --rm -it -v (pwd)/config:/config:ro --hostname catalyst-one --name catalyst-one --shm-size=7553m -p 1935:1935 -p 8080:8080 livepeerci/catalyst:main MistController -c /config/catalyst-dev.json
```
and

```
ffmpeg -re -i ~/testvids/RocketLeague_2h_2sGOP_1080p60.mkv -c copy -f flv 'rtmp://localhost/live/video+eli'
```

Reply:
@iameli I checked and the same command also works after my changes. Am I missing something?